include/boost/beast/http/fields.hpp

```diff
@@ -744,9 +799,21 @@ class basic_fields
     template<class OtherAlloc>
     friend class basic_fields;
 
+    element*
+    new_element(
+        field name,
+        string_view sname,
+        string_view value,
+        error_code& ec);
+
     element&
-    new_element(field name,
-        string_view sname, string_view value);
+    new_element(
```

Comment:
oh wow.. two functions with the same name, one returning a pointer and the other returning a reference? lol... don't you think they should both return pointers?